client/src/components/page/CustomCallback/index.tsx

```diff
@@ -78,7 +78,7 @@ export const UnSuccessfulLogin = withRouter(BaseUnsuccessfulLogin);
 const CustomConnectedAPICallBack: React.FC<RouteComponentProps<RouteParams>> = (props) => {
   return (
     <ConnectedAPICallback
-      LoadingComponent={Loading}
+      LoadingComponent={() => <Spin size="large" />}
```

Comment:
Looks like we have not updated the tests for this

Reply:
@kelvin-muchiri I just checked that, There are no tests related to LoadingComponent, 
Most of the test cases are related to correct rendering and redirections written by @KipSigei .
Not sure, but may be loader test cases are handled in `@onaio/gatekeeper` from where `ConnectedAPICallback` is imported.

Reply:
There is no way of writing the unit test for this, as we can only mock the backend calls based on either the unit test passes or fails, 

Loading state is usually shown when the backend call is in progress.

I dont know a way of that, if you do please suggest, i am open to suggestion on this

Reply:
@kelvin-muchiri

Reply:
@HaseemUlHaq Can we add a snapshot for the component props in the test case where we check if it **renders correctly**? 

I feel this ought to have been done in that particular test case when this test case was written that's why a change in that piece of code did not make any test fail.